packages/channels/botpress-channel-web/src/views/web/convo/index.jsx

```diff
@@ -127,32 +130,32 @@ export default class Convo extends React.Component {
   renderWelcomeMessage() {
     const message = this.props.config.welcomeMsgText || DEFAULT_WELCOME_MESSAGE
 
-    return <div className={style.paragraph}>{message}</div>
+    return <div className={`${style.paragraph} botpress__paragraph`}>{message}</div>
   }
 
   renderHeader() {
     return (
       <div
-        className={style.header}
+        className={`${style.header} botpress__header`}
         style={{
           color: this.props.config.textColorOnForeground,
           backgroundColor: this.props.config.foregroundColor
         }}
       >
-        <div className={style.line}>
-          <div className={style.title}>
+        <div className={`${style.line} botpress__line`}>
+          <div className={`${style.title} botpress__title`}>
             {this.renderAvatar()}
             {this.renderName()}
           </div>
         </div>
-        <div className={style.text}>{this.renderWelcomeMessage()}</div>
+        <div className={`${style.text} botpress__text`}>{this.renderWelcomeMessage()}</div>
```

Comment:
Alternatively if `text` is _generic_ style (like if _all_ text in the webchat is styled with this class) keeping it as is is good.

Reply:
Here's what I mean. If _any text in the webchat_ has some common style (like font-family: Arial) it's fine to call the class `botpress__text`.

But if this class is only used to style the text inside of the _convo_ it should be something like `botpress__convo-text` or `botpress__message-text`